modules/hipaa-conformance-pack/variables.tf

```diff
@@ -0,0 +1,130 @@
+
+variable "AccessKeysRotatedParamMaxAccessKeyAge" {
+  type    = string
+  default = "90"
+}
+
+variable "CloudwatchAlarmActionCheckParamInsufficientDataActionRequired" {
```

Comment:
What we just made these a list of strings? 

The camel case variables is a departure from every other module. I _get_ why you did it, but makes me wonder if we're doing something wrong because this is the first time we've needed to make the decision.